src/Eloquent/HybridRelations.php

```diff
@@ -324,20 +323,6 @@ public function belongsToMany(
         );
     }
 
-    /**
-     * Get the relationship name of the belongs to many.
-     *
-     * @return string
-     */
-    protected function guessBelongsToManyRelation()
```

Comment:
This was introduced as a compatibility layer in [3e26e05b](https://github.com/mongodb/laravel-mongodb/commit/3e26e05b90cf5e207c66e30ea2021ff9ddb16bf9#diff-b815a7989006c7aaa05e7ccda0d141ca8adfa9c2d7d28bef4d38be00b772fd35R260-R267)